docker-compose/docker-compose-injistack/config/certify-mock-identity.properties

```diff
@@ -0,0 +1,103 @@
+## ------------------------------------------- Mock ID Integration properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.mock.integration
+mosip.certify.integration.audit-plugin=LoggerAuditService
+mosip.certify.integration.vci-plugin=MockVCIssuancePlugin
+
+## ------------------------------------------- Mock ID plugin related properties ------------------------------------------------------------
+mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
+mosip.certify.mock.authenticator.get-identity-url=http://mock-identity-system:8082/v1/mock-identity-system/identity
+mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
+mosip.certify.cache.secure.individual-id=false
+mosip.certify.cache.store.individual-id=true
+# TODO: Onboard secrets for local build
+mosip.certify.mock.vciplugin.issuer.key-cert="dummy-issuer-cert"
+mosip.certify.mock.vciplugin.ca.key-cert=dummy
+mosip.certify.svg-templates=insurance-svg-template.json
+mosip.certify.identifier=http://localhost:8090
+mosip.certify.dataprovider.types={'MockVerifiableCredential','StudentCredential','FarmerProfileCredential'}
+
+mosip.certify.issuer.uri=did:web:vharsh.github.io:DID:harsh
+mosip.certify.issuer.pub.key=did:web:vharsh.github.io:DID:harsh#key-0
```

Comment:
can we please refer to mosip org/repo instead of private one

Reply:
@swatigoel We'll have to create a DID document in a MOSIP repo to do this. Also, a user who is re-running it locally will have to point it to their DID repo.

Reply:
Anyone running it on local will not have access to edit this repo, no point in having it within mosip repos